src/reflow/mod.rs

```diff
@@ -54,6 +54,21 @@ impl Checker for Reflow {
     }
 }
 
+/// Extract line delimiter of a string
+fn extract_delimiter<'s>(s: &'s str) -> &'s str {
```

Comment:
We should not be too simplistic here, the string could contain a mix, in which case we should print a warning and do a majority vote, if equilibrium of two variants, then go with the one that appears earlier (or by means of another definition, `indexmap::IndexMap` would an option for this).